impact.tex

```diff
@@ -305,18 +306,25 @@ \subsubsection{Dissemination and exploitation of results}
 
 \subsubsection{Communication activities}
 
-The main remaining goal for dissemination is making sure that potential users are aware of the tools and services developed by \TheProject.
+The main remaining goal for dissemination is making sure that potential users 
+are aware of the tools and services developed by \TheProject.
 In addition to \TheProject-organised workshops in \taskref{education}{workshops},
 the primary mechanism by which we will communicate our results is through publications and conferences.
 All publications funded by \TheProject will be \textbf{Open Access},
 and sites expecting publications have budgeted funds for paying Open Access fees.
 We will identify and attend appropriate conferences for sharing our work,
-including running tutorials at conferences in historically interested communities such as PyData and SciPy. 
+including running tutorials at conferences in historically interested communities such as PyData and SciPy.
+Also, we will identify and attend conferences from complementary communities such as ROpenSci, Mozilla Science, and Julia
+as well as domain specific conferences to maximise the impact \TheProject and to broaden its audience outside the 
+traditionaly included communities.
+In addition, we will operate a website (\taskref{education}{website})
+for collecting and sharing information about \TheProject and its progress.
 
 In addition, we will operate a website (\taskref{education}{\textbf{website}})
 for collecting and sharing information about \TheProject and its progress. It will provide a centralised 
```

Comment:
It looks like the bit about operating a website has accidentally got duplicated.